src/components/WhoWeAre/Departments.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import Accordion from 'react-bootstrap/Accordion';
+import Masonry from 'react-masonry-css';
+import styled from 'styled-components';
+import { media } from 'styled-bootstrap-grid';
+import { faAngleDown, faAngleUp } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { colors } from '../../theme/styleVars';
+import bios from './bios';
+import Team from './Team';
+
+const Department = () => {
+  const sections = [bios['board'], bios['operations'], bios['technical']];
```

Comment:
Since all sections does is put the bios object keys into an array, you can just set `const sections = Object.keys(bios).map(s => bios[s]);` to be more dynamic. However, to simplify your title logic below, it might be better to do something like this:

```
const sectionTitles = {
  'board': 'BOARD OF DIRECTORS',
  'operations': 'BUSINESS OPERATIONS',
  'technical': 'SITE DEVELOPMENT TEAM'
};
const sections = Object.keys(bios).map(s => ({ title: sectionTitles[s], bios: bios[s] }));
```

Then, below, your title could be:
```
            <h2 className="section-text">
              {sect.title}
              <FontAwesomeIcon
                className="bod-icon"
                icon={activeID === index ? faAngleUp : faAngleDown}
                pull="right"
                size="lg"
              />
            </h2>
```
And your `sect.map` would turn into `sect.bios.map`.

Reply:
So I tried that at first, but the issue of bios[s] gives an error since it is a No index signature with a parameter of type 'string' was found on type '{ board: string; operations: string; technical: string; }